integration_test/third_party_apps_data/agent/ops-agent/linux/enable_apache

```diff
@@ -43,4 +42,4 @@ logging:
 EOF
 
 sudo service google-cloud-ops-agent restart
-sleep 30
+sleep 60
```

Comment:
Why did the sleep need to be increased? 60s looks good, just wondering if there was a known bug or something

Reply:
Oh, because I removed the customized `collection_interval 30s` from all the configurations that are used by the integration tests. So they'll default to `60s`, which is closer to what most users would do.

In theory our tests should know to wait and retry as well. But waiting for 1 min here does not hurt.